tool-box/Dockerfile

```diff
@@ -1,16 +1,32 @@
-FROM fedora:latest
+FROM registry.access.redhat.com/ubi8
+
+MAINTAINER Aly Ibrahim<aly@redhat.com>
 
 ENV OC_CLIENT_MIRROR https://mirror.openshift.com/pub/openshift-v3/clients/3.11.127/linux/oc.tar.gz
 ENV HOME /home/tool-box
 
-RUN dnf -y update && \
-    INSTALL_PKGS="git tree vim unzip python-pip jq" && \
-    dnf -y install $INSTALL_PKGS && \
-    dnf clean all && \
-    curl $OC_CLIENT_MIRROR | tar -C /usr/local/bin/ -xzf - && \
-    mkdir -m 775 $HOME && \
-    chmod 775 /etc/passwd && \
-    pip install git+https://github.com/ansible/ansible.git@stable-2.6
+RUN yum -y update && \
+    INSTALL_PKGS="git vim unzip python36" && \
```

Comment:
Is `tree` not available in the UBI repository?

Reply:
I searched for tree, but couldn't find in the public UBI repository.
Let me know if there is any alternative.